backend/src/v2/driver/util.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2021-2024 The Kubeflow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package driver
+
+import (
+	"fmt"
+	"github.com/kubeflow/pipelines/api/v2alpha1/go/pipelinespec"
+	"regexp"
+)
+
+// InputPipelineChannelPattern define a regex pattern to match the content within single quotes
+// example input channel looks like "{{$.inputs.parameters['pipelinechannel--val']}}"
+const InputPipelineChannelPattern = `\$.inputs.parameters\['(.+?)'\]`
```

Comment:
Why is this exported? As far as I can see, it's only used in this file.

Reply:
you're right, fixed!